cluster/v1beta1/types_placement.go

```diff
@@ -215,6 +221,69 @@ type AddOnScore struct {
 	ScoreName string `json:"scoreName"`
 }
 
+// SpreadPolicy defines how the placement decision should be spread among the ManagedClusters.
+type SpreadPolicy struct {
+	// SpreadConstraints defines how the placement decision should be distributed among a set of ManagedClusters.
+	// The importance of the SpreadConstraintsTerms follows the natural order of their index in the slice.
```

Comment:
I am still not sure if this order is specific to implementation or we should restrict it on API.

Reply:
> I am still not sure if this order is specific to implementation or we should restrict it on API.

In order for clients to have confidence in the portability of their manifests, the behavior needs to be consistent for the API, not just for a particular implementation.